Lacrit/assignment1/q1.cpp

```diff
@@ -37,13 +33,22 @@ void remove_spaces(std::string& str)
 int main() 
 {
 	std::string a, b;
-	bool c; 
+	char case_sensitive; 
+	std::cout << "Input first string: " << std::endl;
 	std::getline(std::cin, a);
+	std::cout << "Input second string: " << std::endl;
 	std::getline(std::cin, b);
-	remove_spaces(a);
-	remove_spaces(b);
-	std::cin >> c;
-	std::cout << isAnagram(a, b, c) << std::endl;
+	std::cout << "Case Sensitive?(c|i) (c => sensitive || i => insensitive)" << std::endl;
+	std::cin >> case_sensitive;
+	const std::string word = "Words aren`t anagrams";
+		remove_spaces(a);
+		remove_spaces(b);	
```

Comment:
I think both remove_space calls and the check if they are the same length should be part of `isAnagram`